Sources/MockoloFramework/Templates/MethodTemplate.swift

```diff
@@ -118,34 +119,41 @@ extension MethodModel {
                 \(2.tab)set { }
                 """
             }
-            
+
             let overrideStr = isOverride ? "\(String.override) " : ""
+            let modifierTypeString: String
+            switch modifier {
+                case .dynamic, .weak:
+                    modifierTypeString = (overrideStr.isEmpty ? "" :  " ") + "dynamic" + " "
```

Comment:
why hardcode "dynamic" here? Also if no need to look up the overrideStr to prepend an extra space here; it's already added to the overrideStr if non-empty.